### ISSUE TITLE
Fix screen reader announcement for file picker component

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1107]
**Description:**
The file picker component on the Data Files page reads "no file selected" even when there is a file selected. This is likely a result of the "no file selected" text not getting passed an ```aria-hidden=true``` property when it is visibly hidden. 

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] "No File selected" is not read on file picker components that **do** have a file 
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] Add aria-hidden=true to "no file selected" text element when text element is not visible
- [ ] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

https://user-images.githubusercontent.com/28515874/156947536-468b497d-4e92-4226-9f23-e8491326e92b.mov

**Comment by valcollignon** [comment]
@reitermb to review this ticket and circle back. per backlog refinement 12.21.21.

**Comment by valcollignon** [comment]
Reviewed briefly in backlog 2.22.22. @reitermb will review.

**Comment by abottoms-coder** [comment]
Discussed in backlog refinement 3.1.22 -- @reitermb to still review. He will test to see if applicable.

**Comment by reitermb** [comment]
Currently getting a 400 error when I try to submit files so I'm blocked on re-testing the success message element of this, but I have confirmed that the no file selected bug still exists, as well as the download button disappearing when clicked.

Added a video and a little refinement to the ticket

**Comment by stevenino** [comment]
@reitermb to split out #1107 into required a11y fix and 2nd tix that can be pushed out to v3.1

**Comment by robgendron** [comment]
Please change to bug format @reitermb

**Comment by lhuxraft** [comment]
Revisit after react scripts is upgraded

**Comment by lhuxraft** [comment]
@reitermb to discuss with Thomas during next a11y check-in

**Comment by reitermb** [comment]
Closeable! Testing post-USWDS bump resulted in correct reads. "no file selected" is no longer read for attached / submitted files. The same (correct) behavior was also demonstrated via Voiceover as well as NVDA in the test video below.

https://github.com/user-attachments/assets/a828da06-5ba2-4f6e-8ed9-8d96883115e6


cc @ttran-hub @ADPennington @victoriaatraft @lhuxraft

**Comment by lhuxraft** [comment]
Thank you @reitermb! Love a good ticket close 😄
